api/oneclick.py

```diff
@@ -800,8 +802,9 @@ def process_availability(self, media_type='ebook'):
             license_pool, is_new, is_changed = self.api.update_licensepool_for_identifier(isbn, available)
             # Log a circulation event for this work.
             if is_new:
-                Analytics.collect_event(
-                    self._db, license_pool, CirculationEvent.DISTRIBUTOR_AVAILABILITY_NOTIFY, license_pool.last_checked)
+                for library in self.collection.libraries:
+                    self.analytics.collect_event(
+                        library, license_pool, CirculationEvent.DISTRIBUTOR_TITLE_ADD, license_pool.last_checked)
```

Comment:
All these places that were calling Analytics.collect_event are missing test coverage.

Also, for some reason this one was using the wrong event type. I think AVAILABILITY_NOTIFY is when a patron has something on hold and it becomes available for checkout. I'm not completely sure what's going on here, but TITLE_ADD is usually the event type for a new license pool.